pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/sqlinstanceauthorizednetworks/create.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: postgresauthorizednetworks-${uniqueId}
+  annotations:
+    cnrm.cloud.google.com/state-into-spec: absent
+spec:
+  databaseVersion: POSTGRES_16
+  region: us-central1
+  settings:
+    ipConfiguration:
+      authorizedNetworks:
+        - name: all
+          value: 0.0.0.0/0
+          expirationTime: 2099-01-01T00:00:00+00:00
+      ipv4Enabled: true
```

Comment:
This field is defaulted to true in the terraform-based controller. Not sure how to handle converting existing resources to the direct controller, because I feel that it should not be defaulted to true. Thoughts / ideas? Is it ok to change the behavior to make it not true by default?

Reply:
Hmm.. actually, it seems like the API defaults to setting `ipv4Enabled: true` if the `ipConfiguration` is not set. So.. maybe it is better behavior to default ipv4Enabled to true actually. Though, not quite sure how to handle this. Maybe in the CRD? Will come back to this on Monday..